spec/rails_helper.rb

```diff
@@ -51,6 +51,14 @@
   # Use the faster rack test by default for system specs if possible
   config.before(:each, type: :system) { driven_by :rack_test }
 
+  # silence warning about "allow[ing] expectations on `nil`" without disabling entirely
+  config.around(:each, :allow_message_expectations_on_nil) do |example|
```

Comment:
I understand what is does... but would be nice to know why we need this?

i.e. what exactly broke that we decided this was the best way to fix?

Reply:
removed as simpler method was implemented